indico/modules/events/templates/display/indico/meeting.html

```diff
@@ -45,13 +50,16 @@ <h1 itemprop="name">
     {% endblock %}
 
     {% block body %}
-        <div class="event-sub-header">
-            <table class="eventDetails">
-                <tbody>
-                    {% include 'events/display/indico/_details.html' %}
-                </tbody>
-            </table>
-        </div>
+        {% if has_subheader %}
+            <div class="event-sub-header">
+                <table class="eventDetails">
+                    <tbody>
+                        {% set hook_event_header = hook_event_header %}
```

Comment:
`{# https://github.com/pallets/jinja/issues/603 #}` to clarify why this is needed